packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -24,10 +26,13 @@ class SparkComponent extends Component {
 
     if (!spark) {
       spark = createSpark(accessToken);
+      metrics.sendLoadStart(spark);
+      metrics.sendElapsedTimeMetric(`sparkObject:loaded`, spark);
       props.storeSparkInstance(spark);
     }
 
     spark.listenToAndRun(spark, `change:canAuthorize`, () => {
+      metrics.sendElapsedTimeMetric(`sparkObject:authenticated`, spark);
```

Comment:
Do we only want to fire this once `spark.canAuthorize` is true? And only fire it once per session?